SportsCave/public/javascripts/clientChat.js

```diff
@@ -0,0 +1,79 @@
+(()=> {
+    var element = (id) =>{
+        return document.getElementById(id);
+    };
+
+    var status = element('status');
+    var messages = element('messages');
+    var textarea = element('textarea');
+    var username = document.getElementById('username').innerHTML;
+    var clearBtn = element('clearBtn');
+
+    var statusDefault = status.textContent;
+
+    var setStatus = (s) =>{
+        status.textContent = s;
+
+        if(s != statusDefault){
+            var delay = setTimeout(() =>{
+                setStatus(statusDefault);
+            }, 4000);
+        }
+    };
+    var socket = io.connect('http://127.0.0.1:3000');
```

Comment:
from when we talked the other day, we might need to make this address the dynamic ip of the machine thats running it.  has anyone tested it?